src/tools/tests/kotlin-schema-field-test.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {assert} from '../../platform/chai-web.js';
+import {Manifest} from '../../runtime/manifest.js';
+import {SchemaGraph} from '../schema2graph.js';
+import {generateFields} from '../kotlin-schema-field.js';
+
+describe('Kotlin Schema Fields', () => {
+  it('generates Kotlin Types', async () => {
+
+    const expectations: [string, string][] = [
```

Comment:
Awesome! I'm glad that there are tests for the new file.

Reply:
They are limited, but still better than what came before.